src/lib/util.ts

```diff
@@ -199,3 +199,35 @@ export function linkToEtherScan(address: Address) {
   }
   return `https://${prefix}etherscan.io/address/${address}`;
 }
+
+export function getClaimableRewards(reward: IRewardState) {
+  if (!reward) {
+    return {};
+  }
+
+  const result: {[key: string]: BN} = {};
+  if (reward.reputationForProposer.gt(new BN(0)) && new BN(reward.reputationForProposerRedeemedAt).isZero()) {
```

Comment:
why are you converting the RedeemedAt fields to BigNumbers? arent they dates?

Reply:
because of a client bug. Now fixed.